<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllBugList</class>
 <widget class="QDialog" name="GetAllBugList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetAllBugList</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Select BugTask Title</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Task Title</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTaskTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected Bug Status</string>
   </property>
   <widget class="QLabel" name="label_BugStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEXT</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected Bug Description</string>
   </property>
   <widget class="QTextEdit" name="textEditBugDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
